extern/guided_diffusion/guided_diffusion/train_util.py

```diff
@@ -33,15 +40,20 @@ def __init__(
         log_interval,
         save_interval,
         resume_checkpoint,
+        val_data=None,
```

Comment:
move it to be one of the last arguments plz

Reply:
Added in new issue

Reply:
https://github.com/talafek96/Diffusion-AD-Project/pull/46